rosdep/python.yaml

```diff
@@ -5240,6 +5240,15 @@ python3-imageio:
   debian: [python3-imageio]
   gentoo: [dev-python/imageio]
   ubuntu: [python3-imageio]
+python3-kitchen:
+  arch: [python-kitchen]
+  debian: [python3-kitchen]
+  fedora: [python3-kitchen]
+  gentoo: [dev-python/kitchen]
+  osx:
+    pip:
+      packages: [kitchen]
+  ubuntu: [python3-kitchen]
```

Comment:
The one problem with this is that `python3-kitchen` doesn't seem to be available on xenial, which is still a supported platform.  I'd suggest using the "null" syntax for xenial, like https://github.com/ros/rosdistro/blob/06a8165d68496e3d7c21da497cab4174d5c20fc1/rosdep/python.yaml#L4699

Reply:
Null syntax for xenial in 0e5a2f63b556510a86fdd81230a93562b8f31856